src/Core.UnitTests/CFamily/EffectiveRulesConfigCalculatorTests.cs

```diff
@@ -165,11 +196,18 @@ public void Cache_MultipleEntries()
 
         #endregion Cache tests
 
-        private static ICFamilyRulesConfig CreateMockConfig(string languageKey)
+        internal const string WellKnownPartialRuleKey1_Active = "rule1";
+        internal const string WellKnownPartialRuleKey2_Active = "rule2";
+        internal const string WellKnownPartialRuleKey3_Inactive = "rule3";
+
+        private static ICFamilyRulesConfig CreateWellKnownRulesConfig(string languageKey)
```

Comment:
Maybe move it to TestInitialize and store it as a member, It's a bit easier IMO to follow it this way.